sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -29,6 +29,15 @@ export function getGenderCode(gender: string): string {
   return gender === "female" ? "F" : "O";
 }
 
+/**
+ * Returns phone number keeping only digits 0 to 9.
+ * @param phone phone number to be formatted.
+ * @returns the phone number. (Eg: +1 (788) 999-7777 => 17889997777)
```

Comment:
As a sentence I believe that it should be written as 
```
the phone number (e.g. +1 (788) 999-7777 => 17889997777).
```

Reply:
Changed it.